components/Panel/VotePanel/Details.tsx

```diff
@@ -43,12 +43,12 @@ export function Details({
   augmentedData,
   assertionChildChainId,
   assertionAsserter,
-  claim,
```

Comment:
turns out claim is already provided through a serverless call to get augmented data, so we prefer that, vs this tacked on data which doesnt seem to be workign